src/js/components/TimeAgo.js

```diff
@@ -73,7 +73,7 @@ class TimeAgo extends React.Component {
       <time
         title={DateUtil.msToUTCDate(time)}
         dateTime={DateUtil.msToUTCDate(time)}
-        {...Util.omit(this.props, ["prefix", "suppressSuffix", "time"])}
+        {...Util.omit(this.props, Object.keys(TimeAgo.propTypes))}
```

Comment:
This is a good question, I was just following a pattern, applying it to all the cases. I'll research how we're using the `TimeAgo` component and if we don't pass anything except what is in the propTypes I'd say we remove the line 👍